tests/unit_tests/feature_extractors/detection/test_bounding_boxes_area.py

```diff
@@ -18,28 +18,28 @@ def test_compute_histogram(self):
             }
         )
 
-        result = DetectionBoundingBoxArea._compute_histogram(test_df, transform_name="sqrt")
+        result = DetectionBoundingBoxArea._compute_histogram(test_df, name="bbox_area_sqrt")
 
         expected_result = {
-            "train": {"transform": "sqrt", "bin_width": 1, "max_value": 3, "histograms": {"A": [0, 1, 0, 2], "B": [0, 0, 1, 0]}},
-            "val": {"transform": "sqrt", "bin_width": 1, "max_value": 3, "histograms": {"A": [0, 0, 1, 0], "B": [0, 0, 0, 1], "C": [0, 0, 0, 1]}},
+            "train": {"transform": "bbox_area_sqrt", "bin_width": 1, "max_value": 3, "histograms": {"A": [0, 1, 0, 2], "B": [0, 0, 1, 0]}},
+            "val": {"transform": "bbox_area_sqrt", "bin_width": 1, "max_value": 3, "histograms": {"A": [0, 0, 1, 0], "B": [0, 0, 0, 1], "C": [0, 0, 0, 1]}},
```

Comment:
Could you add tests for the `bbox_area_perimeter_sqrt ` as well ?

Reply:
added